sui-execution/src/verifier.rs

```diff
@@ -39,4 +39,42 @@ pub trait Verifier {
 
         self.meter_compiled_modules(protocol_config, &modules)
     }
+
```

Comment:
this is the code added to allow changes in verification runs. 
We can expand `VerifierOverrides` in time as needed.
I think this is ok, but don't know if people have strong feelings about it